packages/omgidl-serialization/src/DeserializationInfoCache.ts

```diff
@@ -269,6 +382,43 @@ export const PRIMITIVE_ARRAY_DESERIALIZERS = new Map<string, ArrayDeserializer>(
   ["string", readStringArray],
 ]);
 
+export const PRIMITIVE_DEFAULT_VALUE_GETTERS = new Map<string, () => unknown>([
```

Comment:
Can that be a `<string, unknown>` map instead?

Reply:
It can, but because this is how the default values are accessed by complex types as well I think it's preferable to keep them consistent.